lib/source-map-generator.js

```diff
@@ -259,6 +259,18 @@ SourceMapGenerator.prototype.applySourceMap =
 SourceMapGenerator.prototype._validateMapping =
   function SourceMapGenerator_validateMapping(aGenerated, aOriginal, aSource,
                                               aName) {
+    // When aOriginal is truthy but has empty values for .line and .column,
+    // it is most likely a programmer error. In this case we throw a very
+    // specific error message to try to guide them the right way.
+    // For example: https://github.com/Polymer/polymer-bundler/pull/519
+    if (aOriginal && typeof aOriginal.line !== 'number' && typeof aOriginal.column !== 'number') {
```

Comment:
I wonder if this should check whether either is not a number, like
```
aOriginal && (typeof ... != 'number' || typeof ... != 'number')
```

I wasn't sure.  Let me know what you think; I'll either merge this patch or the obvious follow-up.